back/app/policies/idea_policy.rb

```diff
@@ -46,7 +46,7 @@ def create?
   end
 
   def show?
-    return false if record.participation_method_on_creation.never_show?
+    return false if !record.draft? && record.participation_method_on_creation.never_show?
```

Comment:
Does it mean that a phygital survey (draft) can be shown?

Reply:
Yes, phygital surveys can already be shown - they only need to be shown in draft. After that, when published the only way to see any data is through the survey results API which doesn't use this policy.